### PR TITLE
PPTP-1379 Additional info level logging

### PR DESCRIPTION
**Pull request by kiranprakashdigital**
### Description of Work carried through

Brief description of what/why/how.

#### Check list 
 - [ ] `./precheck` was executed (Integration/Component/Unit tests)
 - [ ] `sbt scalafmt test:scalafmt` was executed

### CODE REVIEW

File: app/uk/gov/hmrc/plasticpackagingtax/returns/connectors/TaxReturnsConnector.scala [hunk 1]
```diff
@@ -32,17 +34,28 @@ class TaxReturnsConnector @Inject() (
...
     id: String
   )(implicit hc: HeaderCarrier): Future[Either[ServiceError, Option[TaxReturn]]] = {
     import uk.gov.hmrc.http.HttpReads.Implicits.readOptionOfNotFound
     val timer = metrics.defaultRegistry.timer("ppt.returns.find.timer").time()
     httpClient.GET[Option[TaxReturn]](appConfig.pptReturnUrl(id))
       .andThen { case _ => timer.stop() }
-      .map(resp => Right(resp.map(_.toTaxReturn)))
+      .map {
+        resp =>
+          logger.info(s"Found ppt tax returns for id [$id] had response [${toJson(resp)}]")
```
Comment (SteveSugden) [inline app/uk/gov/hmrc/plasticpackagingtax/returns/connectors/TaxReturnsConnector.scala:48]: Is this the whole "tax return"?   I don't think we should be logging that.

File: app/uk/gov/hmrc/plasticpackagingtax/returns/connectors/TaxReturnsConnector.scala [hunk 2]
```diff
@@ -52,7 +65,11 @@ class TaxReturnsConnector @Inject() (
     val timer = metrics.defaultRegistry.timer("ppt.returns.create.timer").time()
     httpClient.POST[TaxReturn, TaxReturn](appConfig.pptReturnsUrl, payload)
       .andThen { case _ => timer.stop() }
-      .map(response => Right(response.toTaxReturn))
+      .map {
+        response =>
+          logger.info(s"Create ppt tax returns had response [${toJson(response)}]")
```
Comment (SteveSugden) [inline app/uk/gov/hmrc/plasticpackagingtax/returns/connectors/TaxReturnsConnector.scala:70]: Again - is the response the whole return?  Without knowing what is in a return it's hard to comment on whether we should be logging it

File: app/uk/gov/hmrc/plasticpackagingtax/returns/connectors/TaxReturnsConnector.scala (hunk 1 above)
Comment (kiranprakashdigital) [reply app/uk/gov/hmrc/plasticpackagingtax/returns/connectors/TaxReturnsConnector.scala:48]: Though the name suggests tax return it only contains just the weights for the id and meta data. This connectors call returns BE.

Comment (SteveSugden) [reply app/uk/gov/hmrc/plasticpackagingtax/returns/connectors/TaxReturnsConnector.scala:48]: That sounds fair enough then.  Thanks for clarifying.

File: app/uk/gov/hmrc/plasticpackagingtax/returns/connectors/SubscriptionConnector.scala [hunk 3]
```diff
@@ -36,11 +38,20 @@ class SubscriptionConnector @Inject() (
...
+
   def get(
     pptReferenceNumber: String
   )(implicit hc: HeaderCarrier): Future[SubscriptionDisplayResponse] = {
     val timer = metrics.defaultRegistry.timer("ppt.subscription.get.timer").time()
     httpClient.GET[SubscriptionDisplayResponse](appConfig.pptSubscriptionUrl(pptReferenceNumber))
+      .map {
+        response =>
+          logger.info(
+            s"Retrieved subscription for ppt reference number [$pptReferenceNumber] had response [${toJson(response)}]"
```
Comment (paulblakehmrc) [inline app/uk/gov/hmrc/plasticpackagingtax/returns/connectors/SubscriptionConnector.scala:51]: There's too much in this particular response. I'd just log the ppt ref here.

File: app/uk/gov/hmrc/plasticpackagingtax/returns/connectors/TaxReturnsConnector.scala (hunk 1 above)
Comment (paulblakehmrc) [inline app/uk/gov/hmrc/plasticpackagingtax/returns/connectors/TaxReturnsConnector.scala:48]: Likewise for all the responses in this returns connector. Logging the id will be enough for now.

**Review by paulblakehmrc — CHANGES_REQUESTED** [review]

File: app/uk/gov/hmrc/plasticpackagingtax/returns/connectors/SubscriptionConnector.scala (hunk 3 above)
Comment (kiranprakashdigital) [reply app/uk/gov/hmrc/plasticpackagingtax/returns/connectors/SubscriptionConnector.scala:51]: Done

File: app/uk/gov/hmrc/plasticpackagingtax/returns/connectors/TaxReturnsConnector.scala (hunk 1 above)
Comment (kiranprakashdigital) [reply app/uk/gov/hmrc/plasticpackagingtax/returns/connectors/TaxReturnsConnector.scala:48]: Done

Comment (paulblakehmrc) [reply app/uk/gov/hmrc/plasticpackagingtax/returns/connectors/TaxReturnsConnector.scala:48]: This connector hasn't changed - the response needs to be removed from the three log statements. Did it miss being pushed?

**Review by paulblakehmrc — CHANGES_REQUESTED** [review]

**Review by paulblakehmrc — APPROVED** [review]

**Pull request merged**